src/Symfony/Component/Form/FormBuilder.php

```diff
@@ -212,10 +208,9 @@ public function getForm()
     /**
      * {@inheritdoc}
      *
-     * @return FormBuilderInterface[]|\Traversable
+     * @return \Traversable<string, FormBuilderInterface>
```

Comment:
are you gonna back-port all these type fixes or does it not matter? this is mainly to make psalm happy. but we do also check psalm on lower branches right.

Reply:
Yes, I intend to do that.